server/container_restore.go

```diff
@@ -226,6 +226,15 @@ func (s *Server) CRImportCheckpoint(
 		return "", fmt.Errorf("specified sandbox not found: %s: %w", sbID, err)
 	}
 
+	systemCtx, err := s.contextForNamespace(sb.Metadata().Namespace)
+	if err != nil {
+		return "", fmt.Errorf("get context for namespace: %w", err)
+	}
+
+	if systemCtx.SignaturePolicyPath != "" {
+		return "", fmt.Errorf("namespaced signature policy defined for pods in namespace %s; signature validation is currently incompatible with restoring", sb.Metadata().Namespace)
```

Comment:
Nit: make the language of the error simpler and more explicit, perhaps.

```suggestion
		return "", fmt.Errorf("namespaced signature policy defined for pods in namespace %s; signature validation is supported for container restore", sb.Metadata().Namespace)
```

Reply:
done (though I corrected "not supported")

Reply:
@haircommander, ah! Sorry! Thank you!